packages/react-native-reanimated/__tests__/bezier.test.ts

```diff
@@ -8,6 +8,23 @@ import { Bezier } from '../src/Bezier';
  */
 // spell-checker:enable
 
+function splitMix32(seed: number) {
```

Comment:
If it is copied from somewhere, let's add a link to the source of the implementation.